services/dask-sidecar/src/simcore_service_dask_sidecar/computational_sidecar/docker_utils.py

```diff
@@ -247,6 +247,11 @@ async def _parse_container_log_file(  # noqa: PLR0913 # pylint: disable=too-many
             )
 
 
+_MINUTE: Final[int] = 60
+_HOUR: Final[int] = 60 * _MINUTE
+_AIODOCKER_LOGS_TIMEOUT_S: Final[int] = 1 * _HOUR
```

Comment:
THOUGH: perhaps we should add some comments on the heuristics behind the timeouts we hard-code in the code . E.g. "NOTE: this is the timeout i experienced that we need to for this and that to happen"